content/app-dev/create-provider/rust.md

```diff
@@ -1,244 +1,131 @@
+Creating a capability provider involves creating a native executable. All capability provider executables have the same basic requirements:
+* Accept a [Host Data](https://wasmcloud.github.io/interfaces/html/org_wasmcloud_core.html#host_data) structure from `stdin` immediately upon starting the executable. The host data is a base64 encoded JSON object with a trailing newline making it easy to pull from the `stdin` pipe.
+* Accept linkdef messages according to the [RPC protocol](../../../reference/lattice-protocols/rpc)
+* Communicate with actors via rpc messages defined by a capability contract,
+* Respond to periodic health checks
 
-While we can use scaffolding, we'll go through it manually so you can see everything that needs to happen. First, create a new Rust crate:
-
-```shell
-cargo new --lib fakepay-provider
-```
-
-We're going to create a _fake_ implementation of the payments capability provider. For obvious reasons, it will be far easier for us to play around and test if we're not trying to make real credit or bank transactions. Here's what our `Cargo.toml` for this new project should look like (the versions may be slightly newer than what you see here):
-
-```toml
-[package]
-name = "fakepay-provider"
-version = "0.1.0"
-authors = ["wasmcloud Team"]
-edition = "2018"
+Thankfully, our scaffolding and reusable Rust crates take care of the basic plumbing for satisfying these requirements, letting you focus on messages exchanged with the actor, as defined by your capability contract.
 
+### Create the New Project
 
-[features]
-static_plugin = [] # Enable to statically compile this into a host
+Let's create a new provider project
 
-[lib]
-crate-type = ["cdylib", "rlib"]
-
-
-[dependencies]
-chrono = "0.4.19"
-env_logger = "0.8.2"
-log = "0.4.14"
-uuid = { version = "0.8.2", features = ["serde", "v4"] }
-wasmcloud-provider-core = "0.1.1"
-payments-interface = { path = "../payments-interface" }
-wasmcloud-actor-core = "0.2.2"
+```shell
+wash new provider fakepay-provider
 ```
 
-There are a couple of things we've put in here because we know we're going to need them soon and, by convention, we like to build capability providers that can either be loaded by virtue of [provider archive](/reference/host-runtime/provider-archive) files or as a direct dependency in a Rust crate.
+The command above creates a new capability provider project called `fakepay-provider`. Lets change to the newly created directory and start writing some code. The implementation below has all the methods a provider would need to implement to satisfy the capability contract `wasmcloud:example:payments`. You could paste this code into src/main.rs (replace the existing contents of that file). The functions themselves haven't been filled out - that's where your business logic goes. You can see that there is almost no boilerplate here, letting you stay focussed on your business logic.
```

Comment:
Let the developer know here if they need any specific value for Vendor, or if the default works (`Acme`)